.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,3 +1,10 @@
+---
+name: Feature Request
```

Comment:
`name` and `about` seem wrong here. Shouldn't then be something like "Bug Report" and "Report a potential bug in Habitat behavior"?